bindings/python/test/coordinate/frame/providers/iers/test_manager.py

```diff
@@ -76,12 +81,7 @@ def test_get_bulletin_a_array_success (self, manager: Manager):
 
     def test_get_bulletin_a_at_success (self, manager: Manager):
 
-        try:
-            bulletin_a: BulletinA = manager.get_bulletin_a_at(Instant.now() - Duration.days(8.0))
-        except RuntimeError:
-            manager.reset()
-            manager.clear_local_repository()
-            bulletin_a: BulletinA = manager.get_bulletin_a_at(Instant.now() - Duration.days(5.0))
+        bulletin_a: BulletinA = manager.get_bulletin_a_at(Instant.now() - Duration.days(7.0))
```

Comment:
I think that one might break at a specific time of the run, @apaletta3 do you confirm? (might be okay, but just checking)

Reply:
So in this diff, I've basically reset the changes introduced by @apaletta3 in his PR.

Since the tests are passing on `master`, why the need to change them (and remove the context cleanup in the fixture)?

Reply:
> So in this diff, I've basically reset the changes introduced by @apaletta3 in his PR.

Yup yup and agreed that it passes on master but I believe at a certain point in time it might not (just because of the issue/release date of the bulletin A). And that's not a big deal because there might only be one day or a few hours where that test based on `now` does not go through, just wanted to point that out ^^ (aligned on the context clean-up in the fixture though)

Reply:
What I fail to understand though: would the proposed changes in the base PR actually make the test pass 100% of the time?